src/backend/src/modules/pia-intake/pia-intake.service.ts

```diff
@@ -59,7 +61,14 @@ export class PiaIntakeService {
     this.validateUserAccess(user, userRoles, existingRecord);
 
     // update the record with the provided keys
-    await this.piaIntakeRepository.update({ id }, { ...updatePiaIntakeDto });
+    await this.piaIntakeRepository.update(
```

Comment:
missing save_id comparison, and 409 exception handling. Are you planning to do that in a followup PR?

Reply:
Yes, I created this PR for the Update Table ticket.